nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps

```diff
@@ -65,24 +65,49 @@ enterprise/web.core.syntax/external/struts-taglib-1.3.10.jar enterprise/web.stru
-extide/gradle/external/gradle-6.7-bin.zip ide/o.apache.commons.lang/external/commons-lang-2.6.jar
-extide/gradle/external/gradle-6.7-bin.zip ide/slf4j.api/external/slf4j-api-1.7.28.jar
-extide/gradle/external/gradle-6.7-bin.zip java/maven/external/aether-api-1.13.1.jar
-extide/gradle/external/gradle-6.7-bin.zip java/maven.embedder/external/apache-maven-3.6.3-bin.zip
-extide/gradle/external/gradle-6.7-bin.zip platform/api.annotations.common/external/jsr305-3.0.2.jar
-extide/gradle/external/gradle-6.7-bin.zip platform/libs.junit4/external/hamcrest-core-1.3.jar
-extide/gradle/external/gradle-6.7-bin.zip platform/o.apache.commons.codec/external/commons-codec-1.13.jar
-extide/gradle/external/gradle-6.7-bin.zip platform/o.apache.commons.io/external/commons-io-2.6.jar
-extide/gradle/external/gradle-6.7-bin.zip ide/o.eclipse.jgit/external/org.eclipse.jgit-5.7.0.202003110725-r.jar
+extide/gradle/external/gradle-7.3-bin.zip extide/libs.gradle/external/gradle-7.3-bin.zip
+extide/gradle/external/gradle-7.3-bin.zip enterprise/libs.amazon/external/ion-java-1.0.2.jar
+extide/gradle/external/gradle-7.3-bin.zip extide/o.apache.tools.ant.module/external/ant-1.10.8.jar
+extide/gradle/external/gradle-7.3-bin.zip extide/o.apache.tools.ant.module/external/ant-launcher-1.10.8.jar
+extide/gradle/external/gradle-7.3-bin.zip ide/bcpg/external/bcpg-jdk15on-1.64.jar
+extide/gradle/external/gradle-7.3-bin.zip ide/bcprov/external/bcprov-jdk15on-1.64.jar
+extide/gradle/external/gradle-7.3-bin.zip ide/c.google.gson/external/gson-2.8.5.jar
+extide/gradle/external/gradle-7.3-bin.zip ide/c.jcraft.jsch/external/jsch-0.1.55.jar
+extide/gradle/external/gradle-7.3-bin.zip ide/c.jcraft.jzlib/external/jzlib-1.1.3.jar
+extide/gradle/external/gradle-7.3-bin.zip ide/libs.commons_compress/external/commons-compress-1.19.jar
+extide/gradle/external/gradle-7.3-bin.zip ide/o.apache.commons.lang/external/commons-lang-2.6.jar
+extide/gradle/external/gradle-7.3-bin.zip ide/slf4j.api/external/slf4j-api-1.7.28.jar
+extide/gradle/external/gradle-7.3-bin.zip java/maven/external/aether-api-1.13.1.jar
+extide/gradle/external/gradle-7.3-bin.zip java/maven.embedder/external/apache-maven-3.6.3-bin.zip
+extide/gradle/external/gradle-7.3-bin.zip platform/api.annotations.common/external/jsr305-3.0.2.jar
+extide/gradle/external/gradle-7.3-bin.zip platform/libs.junit4/external/hamcrest-core-1.3.jar
+extide/gradle/external/gradle-7.3-bin.zip platform/o.apache.commons.codec/external/commons-codec-1.13.jar
+extide/gradle/external/gradle-7.3-bin.zip platform/o.apache.commons.io/external/commons-io-2.6.jar
+extide/gradle/external/gradle-7.3-bin.zip ide/libs.antlr4.runtime/external/antlr4-runtime-4.7.2.jar
+extide/gradle/external/gradle-7.3-bin.zip platform/libs.junit4/external/junit-4.13.2.jar
+extide/gradle/external/gradle-7.3-bin.zip platform/libs.testng/external/jcommander-1.78.jar
+extide/gradle/external/gradle-7.3-bin.zip ide/o.eclipse.jgit/external/org.eclipse.jgit-5.7.0.202003110725-r.jar
```

Comment:
Just one question do we really have extide/libs.gradle/external/gradle-7.3-bin.zip on the tree as well as extide/gradle/external/gradle-7.3-bin.zip

Reply:
You are right - sometime between I tried to extract the tooling from `gradle-7.3-bin.jar`, but that failed miserably, as the embedded tooling is not identical to the independently runnable jar. So it is downloaded again.

I also checked the other `ignored-overlaps` for gradle again and could reduce the list considerably.

With this cleanup, I squashed the changes together and think now it is ready to be merged. When Travis agrees I'll merge.

Thanks for the check and patience :-)